ui/App.svelte

```diff
@@ -101,6 +121,10 @@
 <NotificationFaucet />
 <Theme />
 
+{#if sessionIsUnsealed}
+  <TransactionCenter />
+{/if}
```

Comment:
The transactions should be globally accessible, so as for the initial question regarding the place where they are "initialized", I find it fine there. The issue really is about z-indexing, and that is fair to discuss indeed. 

@brandonhaslegs how would you go about this? We could make a scale of z-indexes for each of the main screens and have them as global variables to avoid hardcoding them ad-hoc and lose the awareness of how one z-index relates to the rest of the system.

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/975/commits/edc0ad58960ffca564cf79158fecdea10f0622a6